packages/muelu/src/Graph/MatrixTransformation/MueLu_AmalgamationFactory_kokkos_def.hpp

```diff
@@ -80,11 +80,21 @@ namespace MueLu {
 
     RCP<Matrix> A = Get< RCP<Matrix> >(currentLevel, "A");
 
+    /* NOTE: Fullblocksize here represents the number of blocks of whatever storage type is represented by the matrix, specifically
+       the GetFixedBlockSize() which should come from the # PDEs specified on the input deck (or null space dimension on finer levels).
+       So for a point matrix, that's the block size.  
+       
+       For a BlockCrsMatrix, that's the number of blocks of the BlockCrs object which make up a logical block for amalgamation.  Here we 
+       divide the GetFixedBlockSize() by the GetStorageBlockSize().  We've only tested fullblocksize=1 in the BlockCrs case, but in theory
+       you could use a larger number here.
+     */
```

Comment:
Same possible change to comment as above.

Reply:
Got it